### ISSUE TITLE
Support comments

### ISSUE THREAD

**Issue by redbassett** [redbassett/wraptor#1]
Currently, the goal of wraptor is to simply insert newlines after a certain number of characters. Currently this causes the following issue:

``` ruby
# This comment will be wrapped at 50 characters, 
but the second line won't be commented
```

When the newline is added, a comment should be added to the new line:

``` ruby
# This comment will be wrapped at 50 characters, 
# and the second line will be commented
```

**Comment by redbassett** [comment]
Issues to consider:
- The added comment symbols add to line count. While generally not an issue, this could cause some weird behaviours with very short line wrap character counts. An edge case that shouldn't occur, but could cause some weird behaviour is someone is trying to break it…
- What is the behaviour around deleting characters? If a line of text or code without a prepended comment symbol is backspaced to the line return, backspacing once more goes to the previous line. Should a commented line, when backspaced to the comment character automatically delete the comment character and return to the previous line, or should it allow the comment character(s) to be deleted by backspace as if they were manually typed? If so, the mechanism for knowing which comments are to be deleted and which not may take some interesting problem solving.

**Comment by redbassett** [comment]
Issue closed in 662c5fb74dbb12764c6c3f796fa76c43814a5499.

**Comment by santos-j** [comment]
What about languages that use different symbols for commenting? Such as % in Latex?
Can we have the option to define custom comment symbols? 
Or have it defined per language instead of being global? Even if it is a small set of languages
